lib/windows.js

```diff
@@ -68,6 +72,74 @@ const kernel32 = new ffi.Library('kernel32', {
 	QueryFullProcessImageNameW: ['int', ['pointer', 'uint32', 'pointer', 'pointer']]
 });
 
+// Check parent window for SubWindows and find the unique process inside of it.
+function getSubWindowRealProcessPath(activeWindowHandle, processPath) {
+	let newProcessPath;
+
+	const windowProcess = ffi.Callback('bool', ['pointer', 'int32'], hwnd => {
+		// Return the Process ID & Process Handle from the Active Window Handle
+		const [, processHandle] = getProcessIdAndHandle(hwnd);
+
+		if (ref.isNull(processHandle)) {
+			return false; // Failed to get process handle
+		}
+
+		const processPathChild = getProcessPath(processHandle);
+
+		if (processPathChild !== processPath) {
+			newProcessPath = processPathChild;
+			return false;
+		}
+
+		return true;
+	});
+
+	user32.EnumChildWindows(activeWindowHandle, windowProcess, 0);
+
+	if (newProcessPath !== undefined && newProcessPath !== processPath) {
+		processPath = newProcessPath;
+	}
+
+	return processPath;
+}
+
+function getProcessPath(processHandle) {
+	if (ref.isNull(processHandle)) {
+		return [undefined, undefined];
+	}
+
+	// Set the path length to more than the Windows extended-length MAX_PATH length
+	// The maximum path of 32,767 characters is approximate, because the "\\?\" prefix may be expanded to a longer string by the system at run time, and this expansion applies to the total length.
```

Comment:
Without installing dependent packages there are no globals. Realistically I think we just return it to the 66000 that was there before. Realistically we're scope creeping since this was existing code and not relevant to the proposed PR. I agree it should be resolved for performance reasons, but really isn't a make or break for the progress of this feature.

Reply:
I'm not talking about using a global. I'm just saying that the current code comments don't cover everything.

Reply:
> We're scope creeping since this was existing code and not relevant to the proposed PR.

It was taken from existing code, yes. But you're copy-pasting it, which makes it your code too. Just because the previous code was under-documented, doesn't mean we should never strive to do better.